packages/legend-studio/src/components/editor/side-bar/WorkspaceRevisionUpdater.tsx

```diff
@@ -0,0 +1,281 @@
+                  <div className="workspace-revision-updater__header__tabs">
+                    {openMergedEditorStates.map((editorState) => (
+                      <div
+                        key={editorState.uuid}
+                        className={clsx(
+                          'workspace-revision-updater__header__tab',
+                          {
+                            'workspace-revision-updaterl__header__tab--active':
+                              editorState === currentDiffEditorState,
+                          },
+                        )}
+                        onMouseUp={closeTabOnMiddleClick(editorState)}
+                      >
+                        <button
+                          className="workspace-revision-updater__header__tab__label"
+                          tabIndex={-1}
+                          onClick={openTab(editorState)}
+                          title={editorState.headerName}
+                        >
+                          {editorState.headerName}
+                        </button>
+                        <button
+                          className="workspace-revision-updater__header__tab__close-btn"
+                          onClick={closeTab(editorState)}
+                          tabIndex={-1}
+                          title={'Close'}
+                        >
+                          <FaTimes />
+                        </button>
+                      </div>
+                    ))}
+                  </div>
+                </div>
+                <div className="panel__content">
+                  {updateConflictState.currentDiffEditorState instanceof
+                    EntityChangeConflictEditorState && (
+                    <EntityChangeConflictEditor
+                      conflictEditorState={
+                        updateConflictState.currentDiffEditorState
+                      }
+                    />
+                  )}
+                  {updateConflictState.currentDiffEditorState instanceof
+                    EntityDiffViewState && (
+                    <EntityDiffView
+                      entityDiffViewState={
+                        updateConflictState.currentDiffEditorState
+                      }
+                    />
+                  )}
+                </div>
+              </div>
+            </ResizablePanel>
+          </ResizablePanelGroup>
+        </div>
+
+        <div className="modal__footer">
+          <button
+            type="button"
+            className="btn btn--primary u-pull-right btn--dark"
+            disabled={Boolean(conflicts.length)}
+            title="Apply Resolutions"
+            onClick={applyResolutions}
+          >
+            Apply Resolutions
+          </button>
+          <button
+            type="button"
+            title="Aborts update"
+            className="btn u-pull-right btn--dark"
+            onClick={abort}
+          >
+            Abort
```

Comment:
Maybe it's still okay if we keep the `Force Update` button around.

Reply:
No they have an option to force update all their changes before reaching conflict resolution. Once in conflict what does force update mean? (i.e if they resolved half their conflicts ?)  Feel like it adds unnecessary logic.

Reply:
Will resolve this thread, I back out from my push for `unified syncing behavior`